llvm/lib/Target/X86/X86InstrSSE.td

```diff
@@ -5737,6 +5755,13 @@ multiclass avx_bittest<bits<8> opc, string OpcodeStr, RegisterClass RC,
             Sched<[sched.Folded, sched.ReadAfterFold]>, VEX;
 }
 
+// ptest is commutable if only the Z flag is used. If the C flag is used,
```

Comment:
`testp`

Reply:
I'd spell it `testps/testpd` or `testpX`